lib/resin-jetson-flash.js

```diff
@@ -177,12 +173,13 @@ module.exports = class ResinJetsonFlash {
                         },
 			['jetson-xavier-nx-devkit']: {
                                 url:
-                                        'https://developer.nvidia.com/downloads/remksjetpack-463r32releasev73t186jetsonlinur3273aarch64tbz2',
+                                        'http://127.0.0.1/Jetson_Linux_R35.4.1_aarch64.tbz2',
```

Comment:
@acostach is this file supposed to be downloaded automatically?

I found it at `https://developer.nvidia.com/downloads/embedded/l4t/r35_release_v4.1/release/jetson_linux_r35.4.1_aarch64.tbz2`, is the right step to modify `build.sh` to add this path?

Reply:
@cbecker that should be https://developer.nvidia.com/downloads/embedded/l4t/r35_release_v4.1/release/jetson_linux_r35.4.1_aarch64.tbz2 , or as it is if you want to serve the archive locally using a http server.

Reply:
ok thanks.

To make it work with the docker image I had to change it in a few places, becuase `build.sh` writes the file with another name, see this line:

```dockerfile
RUN wget "$bsp_url" -O "/tmp/L4T_BSP_$device_type.tbz2"  && echo "BSP archive for $device_type saved in /tmp/L4T_BSP_$device_type.tbz2, can be downloaded from http://127.0.0.1/L4T_BSP_$device_type.tbz2"
``

so it'll never be able to fetch that file, because it'd have to be named `L4T_BSP_xxx` instead, or am I missing something?